conference/src/components/CreateSubmissionForm/CreateSubmissionForm.validator.ts

```diff
@@ -0,0 +1,14 @@
+import * as yup from "yup";
+import { maxSubmissionFileSizeMB } from "../../util/Constants";
+
+export const validationSchema = yup.object({
+  title: yup.string().required("Title is required"),
```

Comment:
i guess we need to add .trim() after all '.string()'s in all validators, it is possible to submit form with only empty spaces